middlewares/validation.middleware.js

```diff
@@ -0,0 +1,22 @@
+//validation
+const validateProduct = (req, res, next) => {
+    const { name, description, price } = req.body;
+    if (!name || !description || price == null) {
```

Comment:
항상 비교할때는 strict equality(엄격한 비교)를 사용해주세요. ==는 타입을 변환해서 비교하기 때문에 javascript에서는 사용하지 않는 것이 좋습니다.

```suggestion
    if (!name || !description || price === null) {
```